examples/simple.py

```diff
@@ -33,11 +33,20 @@ async def produce():
         await asyncio.sleep(2)
 
 
-async def main():
+async def start():
     await stream_engine.start()
     await produce()
+
+
+async def shutdown():
     await stream_engine.stop()
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    loop = asyncio.get_event_loop()
```

Comment:
This code is kind of what could go behind a func like `debug_run` with a similar interface as aiorun

Reply:
It could be for simplicity, but if we add such functionality we need to make sure 100% that the extra `func`, for example `debug_run`, is emulating the same behavior as `aiorun`. We will end up with with more code which we need to test, why don't just use `aiorun` directly?. 

I want to emphasize that end user is responsible of running the engine, the engine shouldn't know anything about how it is ran (asyncio direclty, aiorun, etc). 

The idea with the `simple.py`script is to show how the engine can be run without any extra dependencies, but as the documentation will describe, it is not the recommended usage. 

End users must run the engine, in their preferred way and it is their own responsibility to do it.

Reply:
The getting started page is showing this example. Which is what I think end users will end up copy pasteing.
Maybe we can present the installation with a text like this:

````md
## Installation

```
pip install kstreams
```

You will need a worker, we recommend [aiorun](...)

```
pip install aiorun
```
````

Reply:
Perfect!